service/history/queues/executable.go

```diff
@@ -258,7 +259,7 @@ func (e *executableImpl) HandleErr(err error) (retErr error) {
 		return nil
 	}
 
-	if err == consts.ErrTaskRetry {
+	if err == consts.ErrTaskRetry || err == consts.ErrDeleteOpenExec {
 		e.taggedMetricsHandler.Counter(TaskStandbyRetryCounter).Record(1)
```

Comment:
We don't want the `TaskStandbyRetryCounter` metric get emitted when encountering the task not completed error. Probably define a new counter metric for this error.

Reply:
Added a new counter metric.